src/vision/how_to_vision.md

```diff
@@ -0,0 +1,31 @@
+# ❓ How to vision
+
+## How you can help
+
+| When | What |
+| --- | --- |
+| 🛑  **Coming soon** | Participate in discussions and development towards [roadmap] goals |
+| 🛑  **Coming soon** | Take ownership of "help wanted" goals from the [roadmap] |
+
+## Making the vision real
+
+We are currently working towards implementing the macros vision described in this website. On the [roadmap] page, you can get an overview of the major goals that are part of implementing that future and how we have divided up the work. Each of the goals also has several initiatives, and those initiatives have upcoming milestones. If you'd like to participate in an initiative, you can find the appropriate Zulip stream and see if they are looking for help!
```

Comment:
It might be worth updating this to reflect the fact that we haven't finished the vision doc yet, but once we do we'll work on making it a reality.

Reply:
Make sense thanks!